tmt/steps/provision/testcloud.py

```diff
@@ -434,7 +434,7 @@ def start(self) -> None:
             import guestfs  # noqa: F401
         except ImportError:
             match_legacy = re.search(
-                r'(rhel|centos).*-7', self.image_url.lower())
+                r'(rhel|centos).*-(6\.|7\.).*', self.image_url.lower())
```

Comment:
How about 
```suggestion
                r'(rhel|centos)\D+(6\.|7\.).*', self.image_url.lower())
```
This way the 6 or 7 is the first digit we check, I'm not aware of any rhel/centos images which will have any digit in the name before the major version one

Reply:
You're right, updated, thanks!